reference/wire-protocol.md

```diff
@@ -17,11 +17,32 @@ The message body is a sequence of fields as described by the associated schema.
 
 ## Setup
 
-As soon as a connection is established, the client must send to the server a single word containing the protocol version it wishes to use.
+As soon as a connection is established, the client must send to the server a single word containing the protocol version it wishes to use. The current protocol version is 1 (in little-endian representation).
```

Comment:
Would it be easier to just give the actual word?
```suggestion
As soon as a connection is established, the client must send to the server a single word containing the protocol version it wishes to use. The current protocol version is 1 (`00000001`).
```
(If that one is correct :smile: )

Reply:
Nah, not really - let's leave it as is. :)